robotino_bringup/launch/kclhand_launch_new.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
```

Comment:
please rename this file to kclhand.launch

Reply:
done. Remove the old version kclhand.launch file